client/web/web.go

```diff
@@ -932,8 +1091,7 @@ func (s *Server) servePostRoutes(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if !data.UseExitNode.IsZero() && hasExitNodeRoute(routes) {
-		http.Error(w, "cannot use and advertise exit node at same time", http.StatusBadRequest)
-		return
+		return errors.New("cannot use and advertise exit node at same time")
```

Comment:
There any issues with us returning an [internal server error](https://github.com/tailscale/tailscale/pull/10894/files#diff-0ce5cadcf9a369c37784038b24f01a224992c7eaedef0474ac921e344281dbefR540) here when we used to 400, or is that alright?

Reply:
Nope, that should be fine